recipes/iterm2-tools/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "2.3" %}
+
+package:
+  name: iterm2-tools
+  version: {{ version }}
+
+source:
+  fn: iterm2-tools-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/i/iterm2-tools/iterm2-tools-{{ version }}.tar.gz
+  sha1: a49135aa73f45e135d4a27b6693e10afe597dcb6
+
+build:
+  number: 0
+  script: python setup.py install
+  skip:  # [not osx]
```

Comment:
Interesting that `conda-build` doesn't give us a failure here. Normally it is very sensitive to empty dictionaries (or strings where they shouldn't be).